cfgov/cfgov/urls.py

```diff
@@ -381,6 +381,10 @@ def empty_200_response(request, *args, **kwargs):
         r'^(?i)consumer-tools/financial-well-being/',
         include('wellbeing.urls')
     ),
+    url(
+        r'^(?i)about-us/diversity-and-inclusion/',
+        include('diversity_inclusion.urls')
```

Comment:
Could this [`include`](https://docs.djangoproject.com/en/1.11/ref/urls/#include) specify a namespace, e.g. `namespace='diversity_inclusion`? This allows you to keep the d-i URLs simple (like "form_submitted") while avoiding conflicts with other apps, letting you reference them with`diversity_inclusion:form_submitted`.

Reply:
Done in https://github.com/cfpb/cfgov-refresh/pull/5265/commits/a82f1df9e65bb0be103c0aeecfe68b70229b1de1.